api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -1405,17 +1405,17 @@
         "description": "ContainerStatus contains details for the current status of this container.",
         "properties": {
           "containerID": {
-            "description": "Container's ID in the format '<type>://<container_id>'.",
+            "description": "ContainerID is the ID of the container in the format '<type>://<container_id>'.",
```

Comment:
Just for my own education, does the description always need to start with the property name?

Reply:
I don't think it is a requirement, at least not enforced one. I just normalized everything to look neat and be aligned with golang docs practices